File: KHOI-2010/PHANTICH.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
    freopen("PHANTICH.INP", "r", stdin);
    freopen("PHANTICH.OUT", "w", stdout);
    long long x;
    int demChan = 0, demLe = 0;
    long long mn = LLONG_MAX, mx = LLONG_MIN;
    while (cin >> x) {
        if (x % 2 == 0) demChan++;
        else demLe++;
        mn = min(mn, x);
        mx = max(mx, x);
    }
    cout << demLe << " " << demChan << "\n";
    cout << mx << " " << mn;
    return 0;
}

File: DUNG-2010/thamkhao1/PHANTICH.py
a = list(map(int,input().split()))
chan = 0
le = 0
mi = 999999
ma = 0

for i in a:
    if i % 2 == 0:
        chan +=1
    else:
        le +=1
    mi = min(mi,i)
    ma = max(ma,i)
print(chan,le)
print(ma,mi)